.cirrus.yml

```diff
@@ -330,6 +330,8 @@ freebsd_alt_build_task:
         image_family: freebsd-13-4
     setup_script:
         - pkg install -y gpgme bash go-md2man gmake gsed gnugrep go pkgconf zstd
+    lint_script:
+        - gmake golangci-lint
```

Comment:
I just looked at CI times, this is super slow as the freebsd instance only has 2 cores and 4GB memory.

Not really a blocker for anything you did here but we currently have a two stage CI job model (build/validate -> functional tests) so the slowest lint/build task blocks progress of overall CI time.
And we want that because what is the point of triggering all tests if it is not in a good state to be merged as one build/lint failed.

So I wonder if we can add
```
cpu: 4
memory: 8G
```
to the freebsd_instance if cirrus allows that.

If that is not possible we could split the freebsd lint and build into two different tasks I guess.

Reply:
Implemented.

We can also try to use caching, which should sleep up golangci-lint a lot (unless go is updated or some such).

Reply:
OK, I'm caching `~/.cache/golangci-lint` which helps a lot -- linting takes less than 2 minutes instead of ~8 minutes now, and the cache file is a few megabytes only.